custom/pnlppgi/reports.py

```diff
@@ -36,6 +36,20 @@ def update_config(config):
     config.update(dict({get_INFilter_element_bindparam('owner_id', idx): val for idx, val in enumerate(users, 0)}))
 
 
+def users_locations():
```

Comment:
@czue in specification we have that we should get data for only one user group. But we must show all locations for that group, so I must get all location for that group because we can have situation when users doesn't sent data for particular location. .